macros/core/PGbasicmacros.pl

```diff
@@ -33,7 +33,7 @@ BEGIN
 	be_strict;
 }
 
-my $displayMode;
+my $displayMode = '';
```

Comment:
This isn't really needed.  This variable is always initialized on line 61 in any case.  The important thing is that the `$displayMode` in `main` is defined (set on line 80 of PG.pl), or that will override this anyway and make this `$displayMode` variable be undefined.